app/helpers/products_helper.rb

```diff
@@ -1,16 +1,10 @@
 module ProductsHelper
 
   def price_policy_errors(product)
-    @facility_price_groups ||= current_facility.price_groups.count
-    price_policy_count = product.price_policies.current.count
-    error_msg = ""
-    if price_policy_count == 0
+    if product.is_a?(Bundle)
```

Comment:
I struggled a bit with that. I initially considered overriding `Bundles#price_policies` with `has_many :price_policies, through: :products`, but was scared that would have unintended side effects. Ultimately, I was worried about future confusion, and knew that I would want to switch here anyways to display different messages.